### PR TITLE
Replace main button styles with Bootstrap classes. Add the option for a static button label.

### PR DESCRIPTION
**Pull request by jfeinstein10**

### CODE REVIEW

File: isteven-multi-select.css [hunk 1]
```diff
@@ -32,38 +32,39 @@
...
-    -webkit-user-select: none;
-    -moz-user-select: none;
-    -ms-user-select: none;
-    -o-user-select: none;
-    user-select: none; 
-    white-space:normal;
-    background-color: #fff;
-    background-image: linear-gradient(#fff, #f7f7f7);      
-}
+/*.multiSelect > button {*/
```
Comment (shelman) [inline isteven-multi-select.css]: A lot of stuff commented out here. Remove instead?

File: isteven-multi-select.js [hunk 2]
```diff
@@ -515,6 +515,11 @@ angular.module( 'isteven-multi-select', ['ng'] ).directive( 'istevenMultiSelect'
             // refresh button label
             $scope.refreshButton = function() {
 
+		if ( $scope.lang.staticLabel ) {
+		    $scope.varButtonLabel = $sce.trustAsHtml( $scope.lang.staticLabel + '<span class="caret"></span>' );
```
Comment (shelman) [inline isteven-multi-select.js]: Nice! Is the idea to add an html tag with a class in the static label value itself if we want to ellipsize it?

Comment (jfeinstein10) [inline isteven-multi-select.js]: That's one thing that I wasn't even considering (not sure it'd work for that either?). Rather, I'm using it to just specify a label like "Select monkeys" which doesn't change regardless of the selection

File: isteven-multi-select.css (hunk 1 above)
Comment (jfeinstein10) [inline isteven-multi-select.css]: Yep!

**Pull request merged**